src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs

```diff
@@ -65,142 +70,239 @@ public class ProjectFileInfo
             var projectFileInfo = new ProjectFileInfo();
             projectFileInfo.ProjectFilePath = projectFilePath;
 
-            var properties = new Dictionary<string, string>
+            if (!PlatformHelper.IsMono)
```

Comment:
Yes it was: https://github.com/OmniSharp/omnisharp-roslyn/blob/cdfa2493df6fa4cabeb9ee44757f51319829edef/src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.cs#L61